.github/workflows/build-run-applications.yml

```
@@ -151,6 +169,7 @@ jobs:
             target: "esp32s3"
     env:
       TEST_RESULT_NAME: test_results_${{ matrix.runner.target }}_${{ matrix.runner.marker }}_${{ matrix.idf_ver }}
+      BENCHMARK_RESULT_NAME: benchmark_${{ matrix.runner.target }}_${{ matrix.runner.marker }}_${{ matrix.idf_ver }}
       TEST_RESULT_FILE: test_results_${{ matrix.runner.target }}_${{ matrix.runner.marker }}_${{ matrix.idf_ver }}.xml
```

Could we do something like this?

I don't like the handling of `PYTEST_BENCHMARK_IGNORE` env variable. First we set default value and then we conditionaly change it, which is unclear. Can we define the env var in job level?

This way we can also eliminate the `Set ignores` step

```suggestion
      TEST_RESULT_FILE: test_results_${{ matrix.runner.target }}_${{ matrix.runner.marker }}_${{ matrix.idf_ver }}.xml
      PYTEST_BENCHMARK_IGNORE: ${{ 
        (contains(github.event.pull_request.labels.*.name, 'Run benchmark') || 
         contains(inputs.WFType, 'Build + Tests + Benchmark') || 
         github.ref_name == 'master') 
         ? '' 
         : "--ignore='examples/display_lvgl_benchmark'" 
      }}
```

---

I changed it - I am testing it now. When it will be done, I let you know.

---

I found solution for this. One note, there cannot be empty string (I added space). Now it is working. Thank you!